modules/gateway/gateway.go

```diff
@@ -58,7 +59,13 @@ func (g *Gateway) Close() error {
 
 	// save the latest gateway state
 	id := g.mu.RLock()
-	if err := g.saveSync(); err != nil {
+	var err error
+	if build.Release == "testing" {
+		err = g.save()
+	} else {
+		err = g.saveSync()
+	}
```

Comment:
I don't think it does. It used to, but the recent PR that made syncing optional un-did that.